src/components/views/rooms/EventTile.tsx

```diff
@@ -659,23 +659,20 @@ export default class EventTile extends React.Component<IProps, IState> {
 
     private renderThreadLastMessagePreview(): JSX.Element | null {
         const { threadLastReply } = this.state;
-        if (!threadLastReply) {
-            return null;
-        }
-
         const threadMessagePreview = MessagePreviewStore.instance.generatePreviewForEvent(threadLastReply);
 
-        if (!threadMessagePreview || !threadLastReply.sender) {
-            return null;
-        }
-
+        const sender = this.thread.roomState.getSentinelMember(threadLastReply.getSender());
         return <>
-            <MemberAvatar member={threadLastReply.sender} width={24} height={24} className="mx_ThreadInfo_avatar" />
-            <div className="mx_ThreadInfo_content">
-                <span className="mx_ThreadInfo_message-preview">
-                    { threadMessagePreview }
-                </span>
-            </div>
+            { sender && (
+                <MemberAvatar member={sender} width={24} height={24} className="mx_ThreadInfo_avatar" />
```

Comment:
should it not render a default avatar based on the MXID rather than nothing if no sender known?